data/map systems.txt

```diff
@@ -14312,6 +14500,63 @@ system Fscher
 		distance 15894
 		period 10098.0
 
+system Fumalsamakah
+	pos -962.334 301.805
+	government Uninhabited
```

Comment:
Uninhabited systems in human space still have governments.
![image](https://github.com/endless-sky/endless-sky/assets/17688683/271dabe1-d866-4e2a-bcc1-74c5c5ae91b2)
```suggestion
	government Pirate
```